policies/registry.rego

```diff
@@ -0,0 +1,42 @@
+package before_resolution
+
+# This file enforces policies requiring all attributes to be defined within
+# a semantic convention "registry".  This is a naming/structure convention
+# used by semantic conventions.
+
+# Helper to create attribute registry violations.
+attr_registry_violation(violation_id, group_id, attr_id) = violation {
+	violation := {
+		"id": violation_id,
+		"type": "semconv_attribute",
+		"category": "attribute_registry",
```

Comment:
nit: not sure how to use it/if it's important, what should go there

Reply:
These are kind of all up to use to determine. I just made some random things here, but happy to alter them.

E.g. we can *just* use "type" 'attribute_registry" *or* we could use "type" "attribute_group, etc." to denote what the error is against.

Reply:
I made a new name for category here, but agree it's unclear how we should use it.